app/components/concept-page/question-card.hbs

```diff
@@ -43,6 +42,7 @@
       <div
         data-test-question-submitted
         class="p-3 sm:p-4 shadow-sm border rounded {{if this.selectedOptionIsCorrect 'border-green-400 bg-green-50' 'border-gray-300 bg-white'}}"
+        {{did-insert this.fireCorrectAnswerConfetti}}
```

Comment:
```suggestion
        {{did-insert this.handleDidInsertCorrectAnswerEmoji}}
```

1. We usually don't give actions names since over time the actions will evolve to do multiple stuff, and it's easier to just name them by how they're triggered
2. Let's try adding this below, to the 🎉 emoji?

Reply:
The reason to attach to the emoji is that it's on the left, and that's where the user needs to start reading from. Now it fires in the center